lib/dirac_mobius.cpp

```diff
@@ -17,46 +24,27 @@ namespace quda {
     }
 
     if (getVerbosity() > QUDA_VERBOSE) {
-      if (zMobius)
+      if (zMobius) {
         printfQuda("%s: Detected variable or complex cofficients: using zMobius\n", __func__);
-      else
+        errorQuda("zMobius has NOT been fully tested in QUDA.\n");
```

Comment:
An `errorQuda` should not be included in a check against `QUDA_VERBOSE`, it should trigger regardless.